bespokefit_tutorial/bespoke_parameters_showcase.ipynb

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
<p>The very last bit of that last sentence is a bit difficult to read. I don't have much of a recommendation aside from that :) Maybe we can break it into sentences and clarify the "using the OpenFF-Toolkit set it in the protocol" (I think maybe it's missing a comma?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>